qemu/tests/cfg/rng_hotplug.cfg

```diff
@@ -7,6 +7,10 @@
     rng_num = 1
     read_rng_timeout = 360
     no no_virtio_rng
+    test_after_hotplug = rng_bat
+    test_before_hotunplug = rng_bat
```

Comment:
Seems there is no difference between `test_after_hotplug` and `test_before_hotunplug` except the variants name.  They both test reading random numbers in guest. Any new cases to test differently? If no, I'd prefer to use one single variant.  Thanks!

Reply:
Modified.
Thank you